src/formpack/utils/replace_aliases.py

```diff
@@ -263,6 +283,9 @@ def replace_aliases_in_place(content, allowed_types=None):
 
     if settings:
         content['settings'] = dict([
-            (settings_header_columns.get(key, key), val)
+            (
+                kobo_specific_sub(settings_header_columns.get(key, key)),
+                pyxform_aliases.yes_no.get(val, val),
```

Comment:
i'm not sure we should subject the all potential values in the [settings worksheet](https://xlsform.org/en/#settings-worksheet) to this `yes_no` replacement. if it's for `KOBO_LOCK_ALL`, consider performing the replacement on that key alone.

Reply:
Good point 💪